style.css

```diff
@@ -4,33 +4,85 @@
   padding: 0;
   text-decoration: none;
   list-style: none;
+  
 }
 
-header {
+.body-wrapper {
   display: flex;
+  flex-direction: column;
+  min-width: 100%;
+  width: 100vw;
+  height: 100%;
+  margin: 0;
+  padding: 5px 5px;
+  box-sizing: inherit;
+
+}
+
+.navbar {
   width: 100%;
-  height: 4rem;
-  padding: 1rem 1.5rem;
+  padding: 3% 2%;
+  font-size: 20px;
+  position: fixed;
 }
```

Comment:
- [ ] [REQUIRED] Good job so far! Your mobile menu button and other elements are working very well, but you have not designed the mobile menu like a [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). Please recheck the design in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) and give the required style.

![image](https://user-images.githubusercontent.com/63915024/181115496-929910a5-ce73-49d2-8ce2-e0b90c6d3d2b.png)